WeakAuras/Prototypes.lua

```diff
@@ -684,68 +684,6 @@ if WeakAuras.IsClassicOrBCCOrWrath() then
     end
     return result;
   end
-elseif WeakAuras.IsShadowlands() then
-  function WeakAuras.CheckTalentByIndex(index, extraOption)
-    local tier = ceil(index / 3)
-    local column = (index - 1) % 3 + 1
-    local _, _, _, selected, _, _, _, _, _, _, known  = GetTalentInfo(tier, column, 1)
-    if extraOption == 4 then
-      return selected or known
-    elseif extraOption == 5 then
-      return not (selected or known)
-    end
-    if extraOption == 0 or extraOption == 2 then
-      return selected or known
-    else
-      return selected
-    end
-  end
-elseif WeakAuras.IsDragonflight() then
```

Comment:
Ah this was dead code because we don't call CheckTalentByIndex on DF.

Reply:
yup i was going to rewrite a CheckTalentBySpellId instead, but since data is already refreshed automatically for current spec, Private.talentInfo is good enough for enableTest